packages/@lwc/wire-service/src/link-context-event.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: MIT
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
+ */
+
+/**
+ * Event fired by wire adapters to link to a context provider
+ */
+export class LinkContextEvent {
```

Comment:
this is the new event that can be used from a wire adapter to link to a context provider.

cc @diervo, @kevinv11n this is the one that we can restriction in platform.

Reply:
@caridy I don't see anything about a `LinkContextEvent` in the RFC. Is there anything written down about this that I can read?

Reply:
ah, you're right... I forgot to push that other RFC (wire protocol reform), will do tomorrow morning.

Reply:
@davidturissini I have update the RFC with all the details.